.travis.yml

```diff
@@ -37,8 +38,8 @@ matrix:
 install:
   - |
       if [ "$TOXENV" = "pypy3" ]; then
-        export PYPY_VERSION="pypy3.5-5.9-beta-linux_x86_64-portable"
```

Comment:
Do you think "Installing Scrapy" section should be updated, with the new minimal pypy version?

Reply:
Makes sense.

Once we fix the remaining issue, though, we should probably try and find the lowest version where tests pass.

Reply:
I'm not sure we should care too much about supporting old pypy versions; I think it is for advanced users to run Scrapy on pypy, and they could install a recent version anyways, to get best performance & compatibility. That should be fine to pick some version which is fresh enough, and doesn't cause us problems. Of course, knowing a specific pypy version is better, but I'd not put too much effort in figuring it out - if that's easy, let's do it, otherwise - don't bother.

Reply:
PyPy 5.9 continues to work.

I’ve also moved the FAQ entry about supported Python versions to `install.rst`, to avoid duplication. The previous information was accurate, though, so if you have any second thoughts about the documentation changes, we can revert that commit and leave the documentation _as is_.